packages/shared/profiles/reducer.ts

```diff
@@ -117,6 +119,24 @@ export function profileReducer(state?: ProfileState, action?: AnyAction): Profil
           }
         }
       }
+
+    case ADDED_PROFILES_TO_CATALOG:
+      const addedProfiles = (action as AddedProfilesToCatalog).payload.profiles
+      const updatedProfilesState = {}
+      for (const profile of addedProfiles) {
+        updatedProfilesState[profile.userId] = {
+          ...state.userInfo[profile.userId],
+          addedToCatalog: true
```

Comment:
Is this planned to be used somewhere else?

Reply:
It's being used here https://github.com/decentraland/kernel/pull/372/files#diff-bcea93a768b74b3a44000978a9f8e8f2b257395933ac042653b8153f0a1c751fR400